examples/guestbook/README.md

```diff
@@ -383,3 +383,60 @@ kubectl delete -f examples/guestbook/redis-slave-service.json
 kubectl delete pod redis-master # This is the only pod that requires manual removal.
 ```
 
+### Troubleshooting
+
+Guestbook can go wrong for a variety of reasons, which makes it an effective test.  Lets test the web app simply using *curl*, so we can see whats going on.
```

Comment:
Maybe say "the Guestbook example can fail for a variety of reasons..." ?

Also, maybe mention the setup you were using when you ran into these problems, so people can judge how near or far they are from your setup.